client/auth/login.jsx

```diff
@@ -135,7 +162,7 @@ module.exports = React.createClass( {
 							<FormTextInput
 								name="login"
 								ref="login"
-								disabled={ requires2fa || inProgress }
+								disabled={ Boolean( requires2fa ) || inProgress }
```

Comment:
I wasn't sure about the policy, too. I used `!!` initially, but decided to go with `Boolean()` since it states the intent "convert to boolean", and didn't produce a line that alternates `!!` with `||`. Didn't know of the caveat with `new`, though, so I'll change that.